vmdb/app/models/container_service.rb

```diff
@@ -6,6 +6,7 @@ class ContainerService < ActiveRecord::Base
 
   belongs_to  :ext_management_system, :foreign_key => "ems_id"
   has_and_belongs_to_many :container_groups
+  has_many :container_service_port_configs
```

Comment:
I think you're maintaining the `configs` suffix here for consistency with `container_port_configs`. But actually it seems redundant to specify that it's a configuration. I think that the naming of these tables should be `container_ports` and `container_service_ports`.